Makefile

```diff
@@ -1,94 +1,95 @@
 TOP_DIR := $(dir $(abspath $(lastword $(MAKEFILE_LIST))))
-CODK_SW_URL := https://github.com/01org/CODK-A-Software.git
-CODK_SW_DIR := $(TOP_DIR)/software
-CODK_SW_TAG ?= master
-CODK_FW_URL := https://github.com/01org/CODK-A-Firmware.git
-CODK_FW_DIR := $(TOP_DIR)/firmware
-CODK_FW_TAG ?= master
+CODK_ARC_URL := https://github.com/01org/CODK-A-ARC.git
+CODK_ARC_DIR := $(TOP_DIR)/arc
+CODK_ARC_TAG ?= master
+CODK_X86_URL := https://github.com/01org/CODK-A-X86.git
+CODK_X86_DIR := $(TOP_DIR)/x86
+CODK_X86_TAG ?= master
 CODK_BL_URL := https://github.com/01org/CODK-A-Bootloader.git
-CODK_BL_DIR := $(CODK_FW_DIR)/bsp/bootable/bootloader
+CODK_BL_DIR := $(CODK_X86_DIR)/bsp/bootable/bootloader
 CODK_BL_TAG ?= master
 CODK_FLASHPACK_URL := https://github.com/01org/CODK-A-Flashpack.git
 CODK_FLASHPACK_DIR := $(TOP_DIR)/flashpack
 CODK_FLASHPACK_TAG ?= master
 
 CODK_DIR ?= $(TOP_DIR)
-FWPROJ_DIR ?= $(CODK_FW_DIR)/projects/arduino101/
-SWPROJ_DIR ?= $(CODK_SW_DIR)/examples/Blink/
+X86_PROJ_DIR ?= $(CODK_X86_DIR)/projects/arduino101/
+ARC_PROJ_DIR ?= $(CODK_ARC_DIR)/examples/Blink/
 
 help:
 	
 check-root:
 	@if [ `whoami` != root ]; then echo "Please run as sudoer/root" && exit 1 ; fi
 
 install-dep: check-root
-	$(MAKE) install-dep -C $(CODK_SW_DIR)
-	$(MAKE) one_time_setup -C $(FWPROJ_DIR)
+	$(MAKE) install-dep -C $(CODK_ARC_DIR)
+	$(MAKE) one_time_setup -C $(X86_PROJ_DIR)
 
-setup: firmware-setup software-setup
+setup: x86-setup arc-setup
 
-clone: $(CODK_SW_DIR) $(CODK_FW_DIR) $(CODK_BL_DIR) $(CODK_FLASHPACK_DIR)
+clone: $(CODK_ARC_DIR) $(CODK_X86_DIR) $(CODK_BL_DIR) $(CODK_FLASHPACK_DIR)
 
-$(CODK_SW_DIR):
-	git clone -b $(CODK_SW_TAG) $(CODK_SW_URL) $(CODK_SW_DIR)
+$(CODK_ARC_DIR):
+	git clone -b $(CODK_ARC_TAG) $(CODK_ARC_URL) $(CODK_ARC_DIR)
 
-$(CODK_FW_DIR):
-	git clone -b $(CODK_FW_TAG) $(CODK_FW_URL) $(CODK_FW_DIR)
+$(CODK_X86_DIR):
+	git clone -b $(CODK_X86_TAG) $(CODK_X86_URL) $(CODK_X86_DIR)
+	ln -s $(abspath $(CODK_X86_DIR)) $(TOP_DIR)firmware
```

Comment:
Need to properly rename the directories using the method shown here: https://github.com/01org/CODK-A-X86/commit/31b4ea118c7580e2f0ae100b3402c7ee890bd3c2